### PR TITLE
desktop: Internationalize metainfo and desktop file

### PR DESCRIPTION
**Pull request by kjarosh**
The main idea behind translation of metainfo and desktop file:

1. Define the `.in` versions of these files without any translations.
2. Generate a PO template (`.pot`) files from `.in`.
3. Upload these files to Crowdin, but do not commit them.
4. Download translated `.po` files from Crowdin.
5. Apply these translations to the original input files (`.in`), but do not commit them.
6. Save output to the expected paths of metainfo and desktop file (without `.in`).

### CODE REVIEW

**Review by Dinnerbone — APPROVED** [review]
LGTM but I'm mostly trusting that you know this stuff :D

File: desktop/packages/linux/rs.ruffle.Ruffle.desktop.in [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Name=Ruffle
+GenericName=Flash Player
+Comment=Play Flash games & movies
+Icon=rs.ruffle.Ruffle
+Exec=ruffle %u
+MimeType=application/x-shockwave-flash;application/vnd.adobe.flash.movie
+Categories=AudioVideo;Player;Graphics;Viewer;VectorGraphics;Game
+Keywords=flash;swf;player;emulator;rust
```
Comment (torokati44) [inline desktop/packages/linux/rs.ruffle.Ruffle.desktop.in:10]: Is the fact that this value is also offered for translation pre-determined by some of the tools used?

File: desktop/packages/linux/rs.ruffle.Ruffle.metainfo.xml.in [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        <category>Viewer</category>
+        <category>VectorGraphics</category>
+        <category>Game</category>
+    </categories>
+    <keywords>
+        <keyword>flash</keyword>
+        <keyword>swf</keyword>
+        <keyword>player</keyword>
+        <keyword>emulator</keyword>
+        <keyword>rust</keyword>
```
Comment (torokati44) [inline desktop/packages/linux/rs.ruffle.Ruffle.metainfo.xml.in:46]: (same here)

File: desktop/packages/linux/rs.ruffle.Ruffle.desktop.in (hunk 1 above)
Comment (kjarosh) [reply desktop/packages/linux/rs.ruffle.Ruffle.desktop.in:10]: Yes, keywords are translatable in general, the question is whether all of them should be. By browsing Wikipedia I saw that all of them could be translated, or at least transliterated.

**Pull request merged**